### PR TITLE
drop support for Rails < 4

### PR DESCRIPTION
**Pull request by krzysiek1507**

### CODE REVIEW

**Comment by krzysiek1507** [comment]
@seuros

**Comment by krzysiek1507** [comment]
@seuros

**Comment by seuros** [comment]
I will work on this today. Thanks for the ping.

**Comment by krzysiek1507** [comment]
Rebased

File: lib/acts_as_taggable_on/compatibility.rb [hunk 1]
```diff
@@ -1,11 +1,7 @@
 module ActsAsTaggableOn::Compatibility
```
Comment (seuros) [inline lib/acts_as_taggable_on/compatibility.rb]: probably we don't need this module now.

Comment (krzysiek1507) [inline lib/acts_as_taggable_on/compatibility.rb]: @seuros it's used [here](https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/tagger.rb#L18-L23), so if we want to drop this module, we have to change API or extract parser for options

**Comment by krzysiek1507** [comment]
Thanks!

**Comment by seuros** [comment]
@krzysiek1507 Thank you for your contribution

**Pull request merged**